pkg/templates/rancherd-22-addons.yaml

```diff
@@ -349,3 +349,51 @@ resources:
             gcInterval: 360
             inspectInterval: 20
             ovsVSCtlConcurrency: 100
+  - apiVersion: harvesterhci.io/v1beta1
+    kind: Addon
+    metadata:
+      name: descheduler
+      namespace: kube-system
+      labels:
+        addon.harvesterhci.io/experimental: "true"
+    spec:
+      repo: http://harvester-cluster-repo.cattle-system.svc/charts
+      version: << .DESCHEDULER_CHART_VERSION >>
+      chart: descheduler
+      {{- if and .Addons .Addons.descheduler }}
+      enabled: {{ .Addons.descheduler.Enabled }}
+      {{- else }}
+      enabled: false
+      {{- end }}
+      valuesContent: |
+        kind: Deployment
+        image:
+          repository: registry.k8s.io/descheduler/descheduler
+          pullPolicy: IfNotPresent
+        deschedulingInterval: 5m
+        replicas: 1
+        deschedulerPolicy:
+          profiles:
+          - name: default
+            pluginConfig:
+            - name: DefaultEvictor
+              args:
+                ignorePvcPods: false
+                evictLocalStoragePods: true
+            - name: LowNodeUtilization
+              args:
+                evictableNamespaces:
+                  exclude:
```

Comment:
can we please extend the excluded namespaces to also include some more of the default namespaces.

From our support-bundles i noticed we collect info about following namespace

```
  * cattle-dashboards
  * cattle-fleet-system
  * cattle-provisioning-capi-system
  * fleet-local
  * local
  * cattle-fleet-clusters-system
  * cattle-logging-system
  * cattle-system
  * harvester-system
  * longhorn-system
  * cattle-fleet-local-system
  * cattle-monitoring-system
  * default
  * kube-system
  ```

except for `default` should we include the rest in the excluded namespace list?